src/components/Table/Table.jsx

```diff
@@ -44,7 +40,7 @@ const getTable = (data, columns, sortState, sortField, sortClick) => (
                 overflow: `${column.field === 'kills' ? 'visible' : null}`,
```

Comment:
@blukai 
at columns.map, do a check for mediaQuery
```js
columns.map((column, index) => {
if (!column.mediaQuery || !column.mediaQuery(width)) {
  return null;
}
```
see my pr on playerheader or the redux-responsive docs for how to bring width into the table. Do the same thing in TableHeader where we map over columns to create the header row.